.github/ISSUE_TEMPLATE/request_rule_change.yml

```diff
@@ -0,0 +1,78 @@
+name: Request a rule change
+description: Request a rule change for the eslint-plugin-testing-library.
+labels: [enhancement]
+body:
+  - type: input
+    id: what_rule_do_you_want_to_change
+    attributes:
+      label: What rule do you want to change?
+      description: Enter the name of the rule you want to change.
+      placeholder: Enter the name of the rule you want to change.
+    validations:
+      required: true
+
+  - type: dropdown
+    id: rule_change_more_fewer_warnings
+    attributes:
+      label: Does this change cause the rule to produce more or fewer warnings?
+      options:
+        - 'More warnings'
+        - 'Fewer warnings'
+    validations:
+      required: true
+
+  - type: textarea
+    id: change_implementation
+    attributes:
+      label: How will the change be implemented?
+      description: Describe how the new rule will be implemented.
+      placeholder: Describe how the new rule will be implemented.
```

Comment:
I guess we can remove this placeholder since it's the same right? I don't have a better suggestion.

Reply:
```suggestion
```

Let's remove the placeholder then. 😉